Blimp/Parameters.h

```diff
@@ -93,24 +93,22 @@ class Parameters
         k_param_adsb,
         k_param_notify,
 
-        //PID Controllers
-        k_param_pid_vel_xy = 32,
```

Comment:
a comment "32 to 37 was used by vel_xxx etc" would be good

Reply:
Ended up deciding not to move all the parameters yet.